test/withGL.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function withGL(init, fn)
+{
+    if (!PIXI.utils.isWebGLSupported())
+    {
+        return undefined;
+    }
+
+    const renderer = init();
+
+    try
+    {
+        return fn(renderer);
+    }
+    finally
+    {
+        PIXI.utils.destroyTextureCache();
+        renderer.destroy();
```

Comment:
I am not crazy about this abstraction. It obfuscates more than provides a useful utility. Please just add the renderer and destroy functions within the test code. This seems unnecessary. Also you can use the `afterEach` method in Mocha to always destroy the TextureCache.